__tests__/app.test.js

```diff
@@ -54,6 +54,51 @@ describe("/api/articles", () => {
         expect(body).toBeSortedBy("created_at", { descending: true });
       });
   });
+  test("GET:200 - responds with an array of article objects in descending order by created_at(date based on a query with a given topic, ", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toBe(true);
+        expect(body.length).toBe(12);
+        expect(typeof body[0].article_id).toBe("number");
```

Comment:
This would be much more robust if you were to test each of the articles in a forEach, using something like the jest matcher `toHaveProperty`